RELEASE_GUIDELINES.md

```diff
@@ -0,0 +1,59 @@
+# Releasing bittensor
+
+The release manager in charge can release a Bittensor version using the script:
+  - [./scripts/release/release.sh](./scripts/release/release.sh)
+
+## Release script usage
+
+Options:
+  - -V, --version: type of update. It could be major, minor or patch.
+  - -A, --apply: This specify to apply the release. Without this the release will just show a dry run with no changes.
+  - -T,--github-token: A github personal access token to interact with the Github API.
+
+### Github token
+
+Since you need to use a secret when releasing bittensor (github personal access token), I encourage you to use [pass](https://www.passwordstore.org/) or a similar tool that allows you to store the secret safely and not expose it in the history of the machine you use.
+
+So you can have:
+```
+GITHUB_ACCESS_TOKEN=$(pass github/your_personal_token_with_permisions)
+```
+
+or
+```
+GITHUB_ACCESS_TOKEN=$(whatever you need to get the token safely)
+```
+
+### Executions
+
+So, executing the script to release a minor version will be:
+
+```
+# For a dry run
+./scripts/release/release.sh --version minor --github-token $GITHUB_ACCESS_TOKEN`
+```
+
+```
+# Applying changes
+./scripts/release/release.sh --apply --version minor --github-token $GITHUB_ACCESS_TOKEN`
```

Comment:
does the release script automatically push to pip as well? or do we have to do that manually?

Reply:
yes yes, it uses the script [release_pip.sh](https://github.com/opentensor/bittensor/pull/1005/files#diff-b42d8e6474dad13b23bef0ca4659d47cc2c70df68d6a6e0ab2773a458db22477)

Reply:
It's being called in the script [release.sh](https://github.com/opentensor/bittensor/pull/1005/files#diff-cfab82936ab53604a639532581ead5e03e829cc60d793e7eae8de5863e88b20dR155)